Source/Particles/Collision/BinaryCollision/BinaryCollision.H

```diff
@@ -306,6 +306,31 @@ public:
                                                     p_n_pairs_in_each_cell, pair_offsets.data());
             index_type* AMREX_RESTRICT p_pair_offsets = pair_offsets.dataPtr();
```

Comment:
In the tests that I ran, the number of pair-particles is zero so a bunch of code doesn't execute at all. Can't say if there aren't any cases where this will be valid.